spinnakerdatadog/handlers.go

```diff
@@ -44,7 +46,15 @@ func (et *EventTemplate) Compile() error {
 	if err != nil {
 		return errors.Wrap(err, "could not compile eventText")
 	}
-
+	var tags []*template.Template
```

Comment:
eh, might rewrite this to just append directly to `et.compiledTags`. not a fan of this anyway.